frontend/src/App.js

```diff
@@ -26,62 +28,54 @@ const imgStyle = {
 };
 
 let gridItem = {
```

Comment:
replace `let` with `const`. `let` should only be used when variable will be mutated, like index variable in a for loop